tests/testthat/test-spread.R

```diff
@@ -24,7 +24,10 @@ test_that("convert turns strings into integers", {
 test_that("duplicate values for one key is an error", {
   df <- tibble(x = factor(c("a", "b", "b")), y = c(1, 2, 2), z = c(1, 2, 2))
   expect_error(spread(df, x, y),
-    "Duplicate identifiers for rows (2, 3)",
+    paste0("Each row of output must be identified by a unique combination of keys.\n",
```

Comment:
This is a bit long — the match text is a regular expression, so can you please tweak it to just match a reasonable fragment of the full error?